network/dag/state.go

```diff
@@ -65,19 +63,36 @@ func NewState(db stoabs.KVStore, verifiers ...Verifier) (State, error) {
 	graph := newDAG(db)
 
 	payloadStore := NewPayloadStore()
+	transactionCount := transactionCountCollector()
+	err := prometheus.Register(transactionCount)
+	if err != nil && err.Error() != (prometheus.AlreadyRegisteredError{}).Error() { // No unwrap on prometheus.AlreadyRegisteredError
```

Comment:
is this going to work when you start multiple instances in a test?

Reply:
no and it will never will with the way the prometheus client library works. It has one global registry.
The tests for seeing if the counter works are never executed in parallel so will work.

Other tests could have a wrong metric counter, which isn't a big deal